deployment/arc-k8s-extension/ServiceGroupRoot/ScopeBindings/Public.ScopeBindings.json

```diff
@@ -33,7 +37,11 @@
                 {
                     "find": "__ACR_NAME__",
                     "replaceWith": "$(ACRName)"
-                },              
+                },       
```

Comment:
why do we need to replace **twice** _REPO_TYPE value in this file?

Reply:
This file has all regions scopes including canary, pilot, ... so on, for each region scope, need to add the repo type